src/libxively/io/fs/xi_fs_api.h

```diff
@@ -10,47 +10,8 @@
 #include <stdlib.h>
```

Comment:
Now that you have me thinking of copyright dates, please update this file's copyright date.